src/frontend/components/Felleskomponenter/Datovelger/Datovelger.test.tsx

```diff
@@ -2,16 +2,36 @@ import React from 'react';
 
 import { act, render } from '@testing-library/react';
 import { renderHook } from '@testing-library/react-hooks';
-import { mock, mockDeep } from 'jest-mock-extended';
+import dayjs from 'dayjs';
+import { mockDeep } from 'jest-mock-extended';
 import { IntlProvider } from 'react-intl';
 
-import { ISODateString } from '@navikt/familie-form-elements';
-import { Felt, ISkjema, useFelt, Valideringsstatus } from '@navikt/familie-skjema';
+import { ESvar, ISODateString } from '@navikt/familie-form-elements';
+import { ISkjema, useFelt } from '@navikt/familie-skjema';
 
 import { SkjemaFeltTyper } from '../../../typer/skjema';
-import { silenceConsoleErrors, TestProvidere } from '../../../utils/testing';
+import {
+    mekkGyldigSøknad,
+    silenceConsoleErrors,
+    spyOnUseApp,
+    TestProvidere,
+    TestProvidereMedEkteTekster,
+} from '../../../utils/testing';
+import OmDeg from '../../SøknadsSteg/OmDeg/OmDeg';
+import { OmDegSpørsmålId } from '../../SøknadsSteg/OmDeg/spørsmål';
 import Datovelger from './Datovelger';
 
+jest.mock('react-router-dom', () => ({
```

Comment:
Hvorfor er det nødvendig å mocke history bare for å teste datovelgeren?
Hvis det er nødvendig burde vi huske å bruke mockHistory-funksjonen fra testing-utils slik at vi gjør det likt over alt.

Reply:
Bruker om deg komponenten for å teste datovelger sine feilmeldinger. Har nå lagt til å bruke mockhistory funksjonen nå :D